xml/s4s_remoteaccess.xml

```diff
@@ -69,69 +69,24 @@
    </para>
    <step>
     <para>
-     First, create an exception for the firewall. Start by creating a file that
-     sets up the port that needs to be opened for RDP.
-    </para>
-    <para>
-     As <systemitem class="username">root</systemitem>, create a new file
-     under <filename>/etc/sysconfig/SuSEfirewall2.d/services/</filename> with
-     the name <filename>xrdp</filename> and the following content:
-    </para>
-    <screen>
-## Name: Remote Desktop Protocol
-TCP="3389"
-    </screen>
-   </step>
-   <step>
-    <para>
-     Open the file <filename>/etc/sysconfig/SuSEfirewall2</filename> and
-     change the lines for the settings
-     <option>FW_CONFIGURATIONS_EXT</option>,
-     <option>FW_CONFIGURATIONS_DMZ</option>, and
-     <option>FW_CONFIGURATIONS_INT</option> to include
-     <literal>xrdp</literal>. If there are no other services enabled, the
-     respective lines should read:
-    </para>
-    <screen>
-FW_CONFIGURATIONS_EXT="xrdp"
-FW_CONFIGURATIONS_DMZ="xrdp"
-FW_CONFIGURATIONS_INT="xrdp"
-    </screen>
-    <para>
-     If there are other services, separate them within the quotes using a
-     space character.
+     First, create an exception for the firewall. Add the port (3389)
+     the protocol (TCP) and the zone. Depending on your setup, replace the placeholder
+     <replaceable>ZONE</replaceable> with <literal>dmz</literal> (for demilitarized zone),
+     <literal>ext</literal> (external), or <literal>int</literal> (internal):
```

Comment:
Which zones to assign depend on how the user's network is configured. I suggest:

"First, create the necessary exceptions for your firewall, opening port TCP 3389 in all relevant zones. For example, if your internal network uses the `<literal>internal</literal>` zone, use the following command:

`<screen>&prompt.root;<command>firewall-cmd</command> --zone=<replaceable>internal</replaceable> --add-port=3389/tcp</screen>`

This is a temporary assignment for testing the new setting. If you need to change more than one zone, change and test each zone one at a time. When you are satisfied tthat new configuration is correct, make it permanent:"

Then add the permanent/reload commands. FYI, you can list the active zones, and all zones:

`firewall-cmd --get-active-zones`

`firewall-cmd --get-zones`

Can you add a link to the firewall section in the Security Guide? 
https://susedoc.github.io/doc-sle/master/single-html/SLES-security/#sec-security-firewall-firewalld

Reply:
@cjschroder Thank you Carla! :+1:  Much appreciated, this make so much more sense than my text.

I've reworked the procedure and added your suggestions. Also added the link to the Security Guide.

After rewriting this procedure, I found some issue which I'd like to clarify with you:

* The `<replaceable>` inside the `<screen>` seems obsolete to me. The `internal` string is a specific value and not a placeholder anymore, right?
* I had to split your paragraph into two steps. Not sure if that was a good idea.
* After I've reworked the procedure, it contains two steps only. Isn't that a bit too less?
* Would it make sense to add a 3rd step? I'm thinking about to enter some magic command that displays the newly added rule. This could act as a final testing/checking step.

Here is how the current procedure looks like:

![procedure](https://user-images.githubusercontent.com/1312925/102604080-f41af400-4123-11eb-8606-4ed8d1e1cd37.png)

Reply:
That looks great! I officially approve. :)